bootstrap/Manifest.zig

```diff
@@ -0,0 +1,23 @@
+const std = @import("std");
+const Manifest = @This();
+
+version: u8,
+name: [:0]const u8,
+winExecutablePath: [:0]const u8,
+unixExecutablePath: [:0]const u8,
+runInWine: ?bool,
```

Comment:
I *think* we're allowed to do this, but should be tested.

```suggestion
runInWine: bool = false,
```